warp10/src/main/java/io/warp10/standalone/Warp.java

```diff
@@ -154,7 +152,7 @@ public static void main(String[] args) throws Exception {
     boolean enableStreamUpdate = !("true".equals(properties.getProperty(Configuration.WARP_STREAMUPDATE_DISABLE)));
     boolean enableREL = !("true".equals(properties.getProperty(Configuration.WARP_INTERACTIVE_DISABLE)));
 
-    for (String property: REQUIRED_PROPERTIES) {
+    for (String property : REQUIRED_PROPERTIES) {
```

Comment:
Revert to usual syntax used in the code base.
Ditto for lines #177,179,201,421,429